orchagent/aclorch.cpp

```diff
@@ -1067,101 +1067,127 @@ void AclOrch::doTask(Consumer &consumer)
     }
 }
 
-void AclOrch::addAclTable(AclTable &newTable, string table_id)
+bool AclOrch::addAclTable(AclTable &newTable, string table_id)
 {
     sai_object_id_t table_oid = getTableById(table_id);
 
     if (table_oid != SAI_NULL_OBJECT_ID)
     {
-        // table already exists, delete it first
-        if (deleteUnbindAclTable(table_oid) == SAI_STATUS_SUCCESS)
+        // If ACL table exists, remove the table first.
```

Comment:
could you change the comment format to `/**/` since all other places are following that convention.